docker/lr-ref/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM continuumio/miniconda3
```

Comment:
can you explain why this deserves an image, and where it is used?
the only nontrivial package it installs is TRF, and if that is the case, I don't see why it is not named `lr-trf`

Reply:
Ah.  This is an image @kvg added when he started this branch.

Nothing seems to be using this image.  I'll take it out.

Reply:
Yeah, this is my bad. I started a thought on a workflow to prepare reference sequence data for use in our workflows, but 1. committed it to the wrong branch, and 2. never finished it.  I just talked to @jonn-smith about it - I don't think we need to pull it out; it's harmless to merge into `main` (nothing is using it yet).  After that, I'll continue that development in a separate branch at some point.